File: dropwizzard/cart-backend/src/main/java/com/mercadolibre/cart_backend/cart/marshallers/cartMarshaller/CartMarshaller_01_00_00.java
package com.mercadolibre.cart_backend.cart.marshallers.cartMarshaller;

import com.fasterxml.jackson.databind.node.ObjectNode;
import com.mercadolibre.cart_backend.cart.entities.cart.Cart_01_00_00;
import com.mercadolibre.cart_backend.core.MarshallerBase;

/**
 * Created by fvitali on 8/8/16.
 */
public class CartMarshaller_01_00_00 extends MarshallerBase<Cart_01_00_00> {
	@Override
	public ObjectNode getJson(Cart_01_00_00 cart) {
		cart.setNote(cart.getNote() + " (modified with CartMarshaller_01_00_00)");
		cart.setMarshaller("CartMarshaller_01_00_00");

		return getObjectNode(cart);
	}
}

File: node.typescript/cart-backend/src/cart/marshallers/cartMarshaller/cartMarshaller_01_00_00.ts
import Marshaller from '../../../core/contracts/marshaller';
import Cart_01_00_00 from '../../entities/cart/cart_01_00_00';

export default class CartMarshaller_01_00_00 implements Marshaller<Cart_01_00_00> {
    public getJson(entity: Cart_01_00_00): any {
        return entity;
    }
}